src/current/cockroachcloud/create-a-basic-cluster.md

```diff
@@ -46,17 +46,17 @@ You cannot currently remove regions once they have been added.
 
 After creating a multi-region cluster deployed on AWS, you can optionally [set up AWS PrivateLink (Limited Access)]({% link cockroachcloud/network-authorization.md %}#aws-privatelink) so that incoming connections to your cluster from applications or services running in your AWS account flow over private AWS network infrastructure rather than the public internet.
 
-Private connectivity is not available for {{ site.data.products.serverless }} clusters on GCP.
+Private connectivity is not available for {{ site.data.products.basic }} clusters on GCP.
 
 Click **Next: Capacity**.
 
 ## Step 4. Configure cluster capacity
 
 Your cluster's capacity dictates its [resource limits](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/architecture/glossary#resource-limits), which are the maximum amount of storage and RUs you can use in a month. If you reach your storage limit, your cluster will be throttled and you may only be able to delete data. If you reach your RU limit, your cluster will be disabled until the end of the billing cycle unless you raise the limit.
 
-All CockroachDB {{ site.data.products.cloud }} organizations get 50M RUs and 10 GiB of storage for free each month. Free resources can be spent across all CockroachDB {{ site.data.products.serverless }} clusters in an organization. You can set higher resource limits to maintain a high level of performance with larger workloads. You will only be charged for what you use.
+All CockroachDB {{ site.data.products.cloud }} organizations get 50M RUs and 10 GiB of storage for free each month. Free resources can be spent across all CockroachDB {{ site.data.products.basic }} clusters in an organization. You can set higher resource limits to maintain a high level of performance with larger workloads. You will only be charged for what you use.
```

Comment:
This is the second mention of the 50M RUs and 10GiB of storage in this PR -- I wonder if it is worth breaking this content out into an include to keep both places in sync.

Reply:
Yep, -- will handle it in April when I work on content updates